src/sdk/apex-extend.ts

```diff
@@ -164,7 +164,7 @@ Apex.prototype.scan = async function ({
     throw new Error(`Failed to init config file: ${e}`);
   }
 
-  if (gconfig) {
+  if (guardConfig) {
```

Comment:
check for guardConfig but then use gconfig ?

Reply:
good catch